src/dfx/src/lib/builders/motoko.rs

```diff
@@ -239,8 +245,10 @@ impl MotokoParams<'_> {
             BuildTarget::Debug => cmd.args(&["-c", "--debug"]),
         };
         cmd.arg("--idl").arg("--stable-types");
-        // TODO add a flag in dfx.json to opt-out public interface
-        cmd.arg("--public-metadata").arg(CANDID_SERVICE);
+        if self.candid_service_metadata_visibility == MetadataVisibility::Public {
+            // private is the default
```

Comment:
Is this comment right? I thought from reading e2e and config.rs that it's public by default

Reply:
This means that `moc`'s default, if you don't pass `--public-metadata`, is to make it private.  I'll make this comment more clear.